components/Note/src/index.scss

```diff
@@ -9,15 +9,10 @@
   line-height: var(--denhaag-note-line-height, 1.5);
   gap: var(--denhaag-note-icon-spacing, var(--denhaag-space-xs));
   padding-block: var(--denhaag-note-padding-block, var(--denhaag-space-2xs));
-}
 
-.denhaag-note__icon,
```

Comment:
Deze CSS selector gebruiken we niet eens..

Reply:
Ik zag de class wel staan in **components/Link/src/stories/bem.stories.mdx**

Reply:
Dubbel gechecked, zie hem nergens staan meer ;-)